cdap-docs/developers-manual/source/building-blocks/workflows.rst

```diff
@@ -16,49 +16,55 @@ transferred to the next job in sequence until the last job in the sequence is ex
 failure, the execution is stopped at the failed job and no subsequent jobs in the sequence
 are executed.
 
-To process MapReduce or Spark programs in sequence, specify
-``addWorkflow()`` in your application::
+To execute MapReduce or Spark programs in the Workflow, you will need to add them in your
+application along with the Workflow. You can optionally add schedule to the Workflow.::
 
   public void configure() {
     ...
-    addWorkflow(new PurchaseHistoryWorkflow());
+    addMapReduce(new MyMapReduce());
+    addMapReduce(new AnotherMapReduce());
+    addSpark(new MySpark());
+    addWorkflow(new MyWorkflow());
+    scheduleWorkflow("FiveHourSchedule", "0 */5 * * *", "MyWorkflow");
```

Comment:
What about adding a link to that here? I can rebuild the Javadocs and send you a link.